_posts/2019-04-23-chat-about-tpu-5.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "闲话TPU #5 Goole TPU 线下分享交流会"
+categories: "TPU Blog"
+author: "Jun.Li"
+date: 2019-04-23
+---
```

Comment:
Could you please add a teaser picture for the blog?

The format would look like this:

```yml
header:
  teaser: /assets/2019/chat-about-tpu-5-group-photo.jpeg
```